### PR TITLE
bug/1074_disable_yafs_avoid_crash_conf_reload

### PR DESCRIPTION
**Pull request by frbattid**
* Fixes issue #1074

### CODE REVIEW

File: doc/cygnus-common/installation_and_administration_guide/running_as_process.md [hunk 1]
```diff
@@ -21,3 +21,4 @@ The parameters used in these commands are:
 * `-p` (or `--mgmt-if-port`). Configures the listening port for the Management Interface. If not configured, the default value is used, `8081`.
 * `-g` (or `--web-app-port`). Configures the port where the web application for Cygnus runs. If not configured, the default value is used, `8082`.
 * `-t` (or `--polling-interval`). Configures the polling interval (seconds) when the configuration is periodically reloaded. If not configured, the default value is used, `30`.
+* `--no-yafs`. Disables Cygnus' YAFS (Yet Another Flume Supervisor), a supervisor in charge of starting an ordered stop of Cygnus when some thread is found dead.
```
Comment (fgalan) [inline doc/cygnus-common/installation_and_administration_guide/running_as_process.md]: Cross-link to yafs.md could be included here.

File: doc/cygnus-ngsi/installation_and_administration_guide/running_as_process.md [hunk 2]
```diff
@@ -21,3 +21,4 @@ The parameters used in these commands are:
 * `-p` (or `--mgmt-if-port`). Configures the listening port for the Management Interface. If not configured, the default value is used, `8081`.
 * `-g` (or `--web-app-port`). Configures the port where the web application for Cygnus runs. If not configured, the default value is used, `8082`.
 * `-t` (or `--polling-interval`). Configures the polling interval (seconds) when the configuration is periodically reloaded. If not configured, the default value is used, `30`.
+* `--no-yafs`. Disables Cygnus' YAFS (Yet Another Flume Supervisor), a supervisor in charge of starting an ordered stop of Cygnus when some thread is found dead.
```
Comment (fgalan) [inline doc/cygnus-ngsi/installation_and_administration_guide/running_as_process.md]: Cross-link to yafs.md could be included here.

Comment (frbattid) [inline doc/cygnus-ngsi/installation_and_administration_guide/running_as_process.md]: Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1439/commits/b9d3563042de82e61576767c2c4942781a00df69

File: doc/cygnus-common/installation_and_administration_guide/running_as_process.md (hunk 1 above)
Comment (frbattid) [inline doc/cygnus-common/installation_and_administration_guide/running_as_process.md]: Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1439/commits/b9d3563042de82e61576767c2c4942781a00df69

File: doc/cygnus-ngsi/installation_and_administration_guide/running_as_process.md [hunk 3]
```diff
@@ -21,4 +21,4 @@ The parameters used in these commands are:
 * `-p` (or `--mgmt-if-port`). Configures the listening port for the Management Interface. If not configured, the default value is used, `8081`.
 * `-g` (or `--web-app-port`). Configures the port where the web application for Cygnus runs. If not configured, the default value is used, `8082`.
 * `-t` (or `--polling-interval`). Configures the polling interval (seconds) when the configuration is periodically reloaded. If not configured, the default value is used, `30`.
-* `--no-yafs`. Disables Cygnus' YAFS (Yet Another Flume Supervisor), a supervisor in charge of starting an ordered stop of Cygnus when some thread is found dead.
+* `--no-yafs`. Disables Cygnus' [YAFS (Yet Another Flume Supervisor)](../../cygnus-common/installation_and_administration/yafs.md), a supervisor in charge of starting an ordered stop of Cygnus when some thread is found dead.
```
Comment (fgalan) [inline doc/cygnus-ngsi/installation_and_administration_guide/running_as_process.md]: Link seems to be broken.

Comment (frbattid) [inline doc/cygnus-ngsi/installation_and_administration_guide/running_as_process.md]: Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1439/commits/34f35f5fe3021ab8cfb43b7f0dc77f3f0009b882

**Comment by fgalan** [comment]
LGTM

**Pull request merged**